### ISSUE TITLE
Is there any pretrained car keypoints models?

### ISSUE THREAD

**Issue by KleinXin** [DuncanZauss/Keypoint_Communities#2]
thx

**Comment by DuncanZauss** [comment]
Hi @KleinXin ,

There was a problem with the OpenPifPaf version, which I fixed now.
Could you git pull this repo again and re-run ```pip install -r requirements.txt```?

Afterwards these models should be available: "shufflenetv2k16-apollo-24", "shufflenetv2k16-apollo-66", "shufflenetv2k30-apollo-66". We currently do not have a pretrained k30 backbone with 24 keypoints (i.e. "shufflenetv2k30-apollo-24").

In case you want to train such a model, [here is the corresponding command](https://openpifpaf.github.io/plugins_apollocar3d.html#train), where you would need to set the basenet to the k30 backbone and add ```--apollo-use-24-kps``` . If you train such a model I would be happy, if you contact me afterwards so we can add it to the pre-trained models.

**Comment by KleinXin** [comment]
Hi @DuncanZauss ，

    I am trainning shufflenetv2k16 24kps on Apollo3D. After one day trainning, some of the losses become negative, do you think it is reasonable? 

   
![image](https://user-images.githubusercontent.com/13084622/137342361-9569af57-64da-4881-a517-6d5ca89b2dd2.png)


The sript I am using now is

CUDA_VISIBLE_DEVICES=1,2,3 python3 -u -m openpifpaf.train --dataset apollo \
--basenet=shufflenetv2k16 --apollo-square-edge=769 \
--lr=0.00002 --momentum=0.95  --b-scale=5.0 \
--epochs=300 --lr-decay 160 260 --lr-decay-epochs=10  --weight-decay=1e-5 \
--weight-decay=1e-5  --val-interval 10 --loader-workers 16 --apollo-upsample 2 \
--apollo-bmin 2 --batch-size 15 --apollo-use-24-kps \
--apollo-train-annotations=/work/Dataset/ApolloCar3D/train/annotations/apollo_keypoints_24_train.json \
--apollo-val-annotations=/work/Dataset/ApolloCar3D/train/annotations/apollo_keypoints_24_val.json \
--apollo-train-image-dir=/work/Dataset/ApolloCar3D/train/images/train \
--apollo-val-image-dir=/work/Dataset/ApolloCar3D/train/images/val

Three Tesla-V100 GPUs are used.

**Comment by DuncanZauss** [comment]
Hi @KleinXin ,
Yes, negative losses are expected.
The screenshot that you sent, looks fine to me, but if you want you could visualize the loss over the epochs with:
```
python -m openpifpaf.logs <PathToYourLogFile>/yourlogfile.log --show
```
In your training command you use "--basenet=shufflenetv2k16", shouldn't it be "--basenet=shufflenetv2k30"?
I always used a batch size of 8 on two V100 32GB GPUs (i. e. "CUDA_VISIBLE_DEVICES=0,1 " ......... "--batch-size=16"), so if a batch size of 8 also fits on the GPUs memory, I would recommend using "--batch-size=24" for three Tesla V100 GPUs.
